spec/collator.html

```diff
@@ -252,7 +252,7 @@ <h1>CompareStrings ( _collator_, _x_, _y_ )</h1>
         </p>
 
         <emu-note>
-          It is recommended that the CompareStrings abstract operation be implemented following Unicode Technical Standard 10, Unicode Collation Algorithm (available at <a href="https://unicode.org/reports/tr10/">https://unicode.org/reports/tr10/</a>), using tailorings for the effective locale and collation options of _collator_. It is recommended that implementations use the tailorings provided by the Common Locale Data Repository (available at <a href="http://cldr.unicode.org/">http://cldr.unicode.org/</a>).
+          It is recommended that the CompareStrings abstract operation be implemented following Unicode Technical Standard 10, Unicode Collation Algorithm (available at <a href="https://unicode.org/reports/tr10/">https://unicode.org/reports/tr10/</a>), using tailorings for the effective locale and collation options of _collator_. It is recommended that implementations use the tailorings provided by the Common Locale Data Repository (available at <a href="https://unicode.org/cldr/trac/wiki">https://unicode.org/cldr/trac/wiki</a>).
```

Comment:
This actually is a different site.  Sorry.  The link I suggested is for CLDR Trac, and works for bugs, but not the home page.

I would probably just keep the link the way it is, but if you really want, the following link works:

https://sites.google.com/site/cldr/

Reply:
CLDR needs a https link, perhaps we can convince unicode to use ssl.